app/3.0/docs/devguide/custom-css-properties.md

```diff
@@ -449,7 +449,7 @@ if (ShadyCSS) {
 ```
 
 Elements using the legacy API can use the
-[`getComputedStyleValue`](/3.0/docs/api/mixins/Polymer.LegacyElementMixin#method-getComputedStyleValue)
+[`getComputedStyleValue`](/3.0/docs/api/legacy/legacy-element-mixin#LegacyElementMixin#method-getComputedStyleValue)
```

Comment:
should be `/3.0/docs/api/legacy/legacy-element-mixin#LegacyElementMixin-method-getComputedStyleValue`?

Reply:
Fixed.